problem-spring-web-autoconfigure/src/main/java/org/zalando/problem/spring/web/autoconfigure/BasicExceptionHandling.java

```diff
@@ -0,0 +1,14 @@
+package org.zalando.problem.spring.web.autoconfigure;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.zalando.problem.spring.web.advice.ProblemHandling;
+
+/**
+ * Activates Problem library with default behaviour if there's no explicit configuration
+ * in the user of the library
+ */
+@ControllerAdvice
+@ConditionalOnMissingBean(ProblemHandling.class) //only if user doesn't declare their own
```

Comment:
Maybe we should just check for `AdviceTrait` here?

Reply:
done